### PR TITLE
sending time block id

### PR DESCRIPTION
**Pull request by JupitersLB**
FE finished

### CODE REVIEW

File: app/controllers/api/v1/time_blocks_controller.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def invite_to_collab
     skills = SharedSkillsMatcher.new(user, current_user).call
     raise StandardError.new('No shared skills') if skills.blank?
 
-    to_user_time = time_block.time.in_time_zone(user.time_zone)
+    to_user_time = @time_block.time.in_time_zone(user.time_zone)
```
Comment (Naokimi) [inline app/controllers/api/v1/time_blocks_controller.rb:24]: need to change lines 14 and 16 too

Comment (JupitersLB) [reply app/controllers/api/v1/time_blocks_controller.rb:24]: change to what ?

Comment (Naokimi) [reply app/controllers/api/v1/time_blocks_controller.rb:24]: add the `@` before the `time_block`

Comment (JupitersLB) [reply app/controllers/api/v1/time_blocks_controller.rb:24]: ahhh I see 👌

**Pull request merged**